evm/src/cpu/columns.rs

```diff
@@ -7,10 +7,12 @@ pub const IS_CPU_CYCLE: usize = 0;
 // If CPU cycle: The opcode being decoded, in {0, ..., 255}.
 pub const OPCODE: usize = IS_CPU_CYCLE + 1;
 
+pub const MEMORY_DUMMY: usize = OPCODE + 1;
```

Comment:
Seems like you're using the real columns in the CTL now, so is this no longer needed?

Reply:
Oh yes!